src/app/tests/suites/certification/Test_TC_S_2_3.yaml

```diff
@@ -20,99 +20,534 @@ PICS:
+              - name: "SceneID"
+                value: 0x01
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x01
+              - name: "TransitionTime"
+                value: 0x0001
 
     - label:
           "TH sends a GetSceneMembership command to DUT with the GroupID field
           set to G1."
       PICS: S.S.C06.Rsp
-      verification: |
-          DUT sends a GetSceneMembershipResponse command to TH with the Status field set to 0x00 (SUCCESS), the Capacity field set to one of the values (SC0 - 1), 0xfe or null, the GroupID field set to G1, the SceneCount field set to 0x01 and the SceneList field containing only the scene ID 0x01.
-      disabled: true
+      command: "GetSceneMembership"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "Capacity"
+                value: (maxScenes/2) - 1
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneList"
+                value: [0x01]
+
+    - label:
+          "TH configures AC2 on DUT for all implemented application clusters
+          supporting scenes."
+      cluster: "Level Control"
+      PICS: PICS_SDK_CI_ONLY
+      command: "MoveToLevelWithOnOff"
+      arguments:
+          values:
+              - name: "Level"
+                value: 200
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 0
+              - name: "OptionsOverride"
+                value: 0
 
     - label:
           "TH configures AC2 on DUT for all implemented application clusters
           supporting scenes."
-      PICS: S.S.C04.Rsp
       verification: |
-          DUT is configured with AC2 which is different from AC1.
-      disabled: true
+          Is DUT configured with AC2?
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_SKIP_SAMPLE_APP
+      arguments:
+          values:
+              - name: "message"
+                value:
+                    "Please configure AC2 on DUT and enter 'y' after the
+                    configuration is complete"
+              - name: "expectedValue"
+                value: "y"
 
     - label:
           "TH sends a RecallScene command to group G1 with the GroupID field set
           to G1 and the scene ID field set to 0x01."
       PICS: S.S.C05.Rsp
-      verification: |
-          (There is no Status response since this was a groupcast) DUT returns to AC1.
-      disabled: true
+      command: "RecallScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x01
```

Comment:
Somewhere in here should be verification somehow that "DUT returns to AC1", yes?  Followup is fine for this part, as long as it's tracked.